src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -186,7 +193,7 @@ def upgrade():
     op.add_column(
         "privacyexperienceconfig",
         sa.Column(
-            "allow_language_selection", sa.Boolean(), server_default="t", nullable=False
+            "allow_language_selection", sa.Boolean(), server_default="f", nullable=False
```

Comment:
Updating PrivacyExperienceConfig.allow_language_selection to have a default value of False as this is a paid feature